pkg/codegen/nodejs/gen.go

```diff
@@ -85,6 +85,10 @@ type modContext struct {
 	disableUnionOutputTypes bool              // Disable unions in output types.
 }
 
+func (mod *modContext) String() string {
```

Comment:
This makes debugging easier by putting the mod names next to the type in the debugger window.